cypress/integration/7-crypto/crypto.spec.ts

```diff
@@ -14,73 +14,146 @@ See the License for the specific language governing permissions and
+            cy.contains(".mx_AccessibleButton", "Got it").click();
+        });
+    };
+
     beforeEach(() => {
-        cy.startSynapse("default").as('synapse').then(
-            synapse => cy.initTestUser(synapse, "Alice"),
-        );
+        cy.startSynapse("default").then(data => {
+            synapse = data;
+            cy.initTestUser(synapse, "Alice");
+            cy.getBot(synapse, { displayName: "Bob", autoAcceptInvites: false }).then(_bob => {
+                bob = _bob;
+            });
+        });
     });
 
     afterEach(() => {
-        cy.get<SynapseInstance>('@synapse').then(synapse => cy.stopSynapse(synapse));
+        cy.stopSynapse(synapse);
     });
 
-    it("should receive and decrypt encrypted messages", () => {
-        cy.get<SynapseInstance>('@synapse').then(synapse => cy.getBot(synapse, "Beatrice").as('bot'));
-
-        cy.createRoom({
-            initial_state: [
-                {
-                    type: "m.room.encryption",
-                    state_key: '',
-                    content: {
-                        algorithm: "m.megolm.v1.aes-sha2",
-                    },
-                },
-            ],
-        }).as('roomId');
-
-        cy.all([
-            cy.get<MatrixClient>('@bot'),
-            cy.get<string>('@roomId'),
-            cy.window(),
-        ]).then(([bot, roomId, win]) => {
-            cy.inviteUser(roomId, bot.getUserId());
-            cy.wrap(
-                waitForEncryption(
-                    bot, roomId, win,
-                ).then(() => bot.sendMessage(roomId, {
-                    body: "Top secret message",
-                    msgtype: "m.text",
-                })),
-            );
-            cy.visit("/#/room/" + roomId);
+    it("setting up secure key backup should work", () => {
+        cy.openUserSettings("Security & Privacy");
+        cy.contains(".mx_AccessibleButton", "Set up Secure Backup").click();
+        cy.get(".mx_Dialog").within(() => {
+            cy.contains(".mx_Dialog_primary", "Continue").click();
+            cy.get(".mx_CreateSecretStorageDialog_recoveryKey code").invoke("text").as("securityKey");
+            cy.contains(".mx_AccessibleButton", "Copy").click();
+            cy.contains(".mx_Dialog_primary:not([disabled])", "Continue").click();
+            cy.contains(".mx_Dialog_title", "Setting up keys").should("exist");
+            cy.contains(".mx_Dialog_title", "Setting up keys").should("not.exist");
         });
+        return;
+    });
 
-        cy.get(".mx_RoomView_body .mx_cryptoEvent").should("contain", "Encryption enabled");
-
-        cy.get(".mx_EventTile_body")
-            .contains("Top secret message")
-            .closest(".mx_EventTile_line")
-            .should("not.have.descendants", ".mx_EventTile_e2eIcon_warning");
+    it("creating a DM should work, being e2e-encrypted / user verification", () => {
+        cy.bootstrapCrossSigning();
```

Comment:
To test verification this is enough (instead of the entire secure backup setup before).

I kept the secure backup as a separate test.